.storybook/blocks/ThemeContainer.jsx

```diff
@@ -1,20 +1,9 @@
 import { DocsContext } from "@storybook/blocks";
-import { styled, ThemeProvider } from "@storybook/theming";
+import { ThemeProvider } from "@storybook/theming";
 import React, { useContext } from "react";
+import { Container } from "./Layouts.jsx";
 
-import "@spectrum-css/tokens/dist/index.css";
-
-const Container = styled.section`
-    color: var(--spectrum-neutral-content-color-default);
-    background-color: var(--spectrum-background-layer-1-color);
-    display: ${props => props.display ?? "flex" };
-    padding-inline: 48px 24px;
-    padding-block: 60px;
-    flex-direction: column;
-    align-items: flex-start;
-    gap: 60px;
-    border-radius: 16px;
-`;
+import tokenStyles from "@spectrum-css/tokens/dist/index.css?raw";
```

Comment:
What does the `?raw` part mean here? I've seen that in a few places and I don't quite know what that is for.

Reply:
Good question! Vite uses this syntax to provide the CSS in a "raw string" format so you can attach the styles using a variable. https://vite.dev/guide/assets#importing-asset-as-string